02-research_team.Rmd

```diff
@@ -0,0 +1,36 @@
+# Research Team
+
+This project is coordinated by the Genomics Data Science Community Network (GDSCN). You can read more about the GDSCN and its mission at the network [website](https://www.gdscn.org/home).
+
+## Soil sampling 
+
+Soil sampling for this project was done by both faculty and student volunteers from schools that aren't traditional R1 research institutions. Many of the faculty are also members of the GDSCN.
+
+- **Annandale, VA**: Northern Virginia Community College
+- **Atlanta, GA**: Spelman College
+- **Baltimore, MD**: College of Southern Maryland, Notre Dame College of Maryland, Towson University
+- **Bismark, ND**: United Tribes Technical College
+- **El Paso, TX**: El Paso Community College, The University of Texas at El Paso
+- **Fresno, CA**: Clovis Community College
+- **Greensboro, NC**: North Carolina A&T State University
+- **Harrisonburg, VA**: James Madison University
+- **Honolulu, Hawai'i**: University of Hawai'i at Mānoa
+- **Las Cruces, NM**: Doña Ana Community College
+- **Montgomery County, MD**: Montgomery College, Towson University
+- **Nashville, TN**: Meharry Medical College
+- **New York, NY**: Guttman Community COllege CUNY
+- **Petersburg, VA**: Virginia State University
+- **Seattle, WA**: North Seattle College, Pierce College
+- **Tsaile, AZ**: Diné College
```

Comment:
Could we embed the map from the gdscn website?? 😄

Reply:
Ooo, I love this idea, although this is the list of everyone who has said they _will_ sample and has gotten a kit, not everyone who has submitted metadata, so there are places on this list that have not yet been collected. I can remove those places without sample metadata to make it cleaner